test/functional/tests.py

```diff
@@ -2044,6 +2059,9 @@ def testLastModified(self):
         hdrs = {'If-Modified-Since': last_modified}
         self.assertRaises(ResponseError, file.read, hdrs=hdrs)
         self.assert_status(304)
+        headers = dict(self.env.conn.response.getheaders())
```

Comment:
using the 304 in the tests is fine by me btw- just use the constant in the main code i think